RELEASES.md

```diff
@@ -2,6 +2,7 @@ A brief list of new features and changes introduced with the specified version.
 
 ### Unreleased
 * [Per organizer logger](https://github.com/adomokos/light-service/pull/162)
+* [Fix 'fail_and_return!' not accepting 'error_code' option](https://github.com/adomokos/light-service/issues/154)
```

Comment:
Can you please use the URL for the PR?

Reply:
Got it! Here's it's addressed on this [commit](https://github.com/adomokos/light-service/pull/168/commits/d702958d995465a2f004f56201b16f2b69f5d240)